test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -61,16 +61,27 @@ declare module 'test-case-reporting' {
     durationMs: number,
   }
 
-  export interface TravisReport {
-    successCount: number,
-    failedCount: number,
-    skippedCount: number,
-    results: Array<Result>,
+  export interface TestResultReport {
```

Comment:
Lets wrap all of these interfaces in `namespace Travis`

Reply:
A namespace is a great idea.
Nit: Do we want to use the name `Travis`, or something that is build-tool-independent (`RemoteReports` or something).

Reply:
You're right, `KarmaReporter` makes more sense here for the namespace

Reply:
Done.